dashboard/test/ui/features/learning_platform/version_history.feature

```diff
@@ -0,0 +1,71 @@
+@no_ie
+@no_mobile
```

Comment:
The versions.feature file I used as a starting point doesn't run on mobile, so I assumed this would fail on mobile as well. That said, I can try removing this and see if it works?

Reply:
Up to you.  If we don't test the base version history scenarios on mobile, then we probably don't need to test these scenarios either.

Reply:
Sounds good. Keeping as-is for now in hopes of a smooth build.